src/examples/scala/containers.scala

```diff
@@ -0,0 +1,13 @@
+object containers extends loamstream.LoamFile {
+  val fileIn = store("fileIn.txt").asInput
+  val fileOut1 = store("fileOut1.txt")
+  val fileOut2 = store("fileOut2.txt")
+  
+  drm {
+    cmd"cp $fileIn $fileOut1"
+  }
+  
+  drmWith(imageName = "docker://library/ubuntu:18.04") {
+    cmd"cp $fileIn $fileOut2"
```

Comment:
Since `store` and `path` are used in various examples, but this is the only example I've seen (so far) with a Docker image, if there's anything special that needs to be done to mount volumes/files to the image, that'd be good to show in an example as well.